e2e/_suites/fleet/installers.go

```diff
@@ -287,16 +292,12 @@ func (i *TARPackage) Install(containerName string, token string) error {
 	return nil
 }
 
-// InstallCerts installs the certificates for a TAR package
+// InstallCerts installs the certificates for a TAR package, using the right OS package manager
 func (i *TARPackage) InstallCerts() error {
-	if err := execCommandInService(i.profile, i.image, i.service, []string{"apt-get", "update"}, false); err != nil {
-		return err
-	}
-	if err := execCommandInService(i.profile, i.image, i.service, []string{"apt", "install", "ca-certificates", "-y"}, false); err != nil {
-		return err
-	}
-	if err := execCommandInService(i.profile, i.image, i.service, []string{"update-ca-certificates", "-f"}, false); err != nil {
-		return err
+	if i.OSFlavour == "centos" {
+		return installCertsForCentos(i.profile, i.image, i.service)
+	} else if i.OSFlavour == "debian" {
+		return installCertsForDebian(i.profile, i.image, i.service)
 	}
```

Comment:
can we have a debug message that installation was skipped because of unknown OS

Reply:
Done in 38d068c